docs/diagrams/TagAddSequenceDiagram.puml

```diff
@@ -0,0 +1,100 @@
+@startuml
+!include style.puml
+scale 1600*561
+
+box Logic LOGIC_COLOR_T1
+participant ":LogicManager" as LogicManager LOGIC_COLOR
+participant ":SoConnectParser" as SoConnectParser LOGIC_COLOR
+participant ":TagCommandParser" as TagCommandParser LOGIC_COLOR
+participant ":TagAddCommandParser" as TagAddCommandParser LOGIC_COLOR
+participant "ta:TagAddCommand" as TagAddCommand LOGIC_COLOR
+participant ":CommandResult" as CommandResult LOGIC_COLOR
+end box
+
+box Model MODEL_COLOR_T1
+participant ":Model" as Model MODEL_COLOR
+end box
+
+[-> LogicManager : execute("tag add 1 t/friend")
+activate LogicManager
+
+LogicManager -> SoConnectParser : parseCommand("tag add 1 t/friend")
+activate SoConnectParser
+
+create TagCommandParser
+SoConnectParser -> TagCommandParser
+activate TagCommandParser
+
+create TagAddCommandParser
+TagCommandParser -> TagAddCommandParser
```

Comment:
I believe that this might be incorrect. The parse method should be before the activation of the constructor of the last parser.

Reply:
Done!